packages/workflow-core/src/lib/plugins/external-plugin/ballerine-email-plugin.ts

```
@@ -0,0 +1,77 @@
+  pluginKind: 'template-email';
+  template: ApiEmailTemplates;
+  displayName: string | undefined;
+  stateNames: string[];
+}
+
+export class BallerineEmailPlugin extends BallerineApiPlugin {
+  public static pluginType = 'http';
+
+  constructor(params: IBallerineEmailPluginParams & IBallerineApiPluginParams & IApiPluginParams) {
+    super(params);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)